app/commands/earthquake.rb

```diff
@@ -11,4 +11,7 @@
   update_timers_channel
   event.respond "Earthquake has been registered!"
   BOT.send_message(TIMER_ALERT_CHANNEL_ID, "**EARTHQUAKE**")
+  if defined? EARTHQUAKE_ALERT_CHANNEL_ID
```

Comment:
Probably want to check to make sure this also has a length, since an empty constant will still be defined. Something like `EARTHQUAKE_ALERT_CHANNEL_ID.to_s.length > 0`

Reply:
updated, also stripped it in case of white space